src/main/java/duke/command/DoneCommand.java

```diff
@@ -0,0 +1,38 @@
+package duke.command;
+
+import duke.exceptions.DukeException;
+import duke.maincomponents.Storage;
+import duke.maincomponents.TaskList;
+import duke.maincomponents.Ui;
+import duke.task.Task;
+
+/**
+ * DoneCommand, which marks a task as done when executed
+ */
+public class DoneCommand implements Command {
+    private int taskDoneInt;
+
+    /**
+     * Constructor of DoneCommand
+     * @param i index of task to mark as done
+     */
+    public DoneCommand(int i) {
+        taskDoneInt = i;
+    }
+
+    /**
+     * Executes the Done command, which marks given task as done
```

Comment:
Perhaps you could include an empty line between description and parameter section to comply with the coding standards? I noticed this in several other places as well.